### DISCUSSION TITLE
Sprite Properties on Top

### DISCUSSION THREAD

**Discussion by guest78266** [ScratchAddons/ScratchAddons#7343] [Ideas]
Hello, this is my first idea I'm going to submit.

### **Sprite Properties on Top**
Moves the sprite properties on top. In the settings. It will give scratchers two options for the sprite properties to be positioned (turn on or off)

It can be viewed in the coding area only, until we can add a windowed paint editor.

I saw this feature in Scratch 1.4 and thought it would be useful to implement this feature into Scratch 3 so we can make sprites draggable or undraggable if we want to without any addon.

**Comment by BroJac5246** [comment]
This doesn't sound very useful to me. What's the idea behind it?
